samples/net/cloud/google_iot_mqtt/prj.conf

```diff
@@ -57,3 +57,9 @@ CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MBEDTLS_HEAP_SIZE=56240
 CONFIG_MBEDTLS_USER_CONFIG_ENABLE=y
 CONFIG_MBEDTLS_USER_CONFIG_FILE="user-tls.conf"
+
+CONFIG_CLOUD_CLIENT_ID="projects/<PROJECT_ID>/locations/<REGION>/registries/<REGISTRY_ID>/devices/"
```

Comment:
Ideally, these should be preceding by comment saying something like: "# Fill in values based on your Google IoT Cloud configuration, see README.rst for more details".

Reply:
Do you know if comments actually work in Kconfig files? I know that Kconfig parses comments as `# CONFIG_FOO is not set`. It looks safe and we use it, but I wanted to make sure.